python/ray/serve/tests/test_multiplex.py

```diff
@@ -21,22 +44,26 @@ async def model_load_func(model_id: str):
         assert multiplexer.models == {"1": "1"}
         # Load model2
         await multiplexer.load_model("2")
+        assert multiplexer._push_multiplexed_replica_info is True
```

Comment:
nit
```suggestion
        assert multiplexer._push_multiplexed_replica_info
```

Reply:
we should also have tests that this is not changed when models are not loaded/unloaded

Reply:
done!